### PR TITLE
[zh-cn]: create the translation of `navigator.virtualKeyboard` property

### PR DESCRIPTION
**Pull request by T34-active**

### CODE REVIEW

File: files/zh-cn/web/api/navigator/virtualkeyboard/index.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+{{APIRef("VirtualKeyboard")}}{{SeeCompatTable}}{{SecureContext_Header}}
+
+{{domxref("navigator")}} 接口的 **`virtualKeyboard`** 只读属性返回对 {{domxref("VirtualKeyboard")}} 实例对象的引用。
+
+{{domxref("VirtualKeyboard_API", "虚拟键盘 API", "", "nocode")}} 为开发人员提供了控制应用程序布局的布局，当虚拟键盘出现在平板电脑、手机或其他可能没有硬件键盘的设备上时，该键盘会自动出现或消失。
+
+## 值
+
+通过 {{domxref("VirtualKeyboard")}} 对象，你可以选择退出自动虚拟键盘行为、以编程方式显示或隐藏虚拟键盘，以及获取虚拟键盘的当前位置和大小。
```
Comment (jasonren0403) [inline files/zh-cn/web/api/navigator/virtualkeyboard/index.md:16]:
```suggestion
一个 {{domxref("VirtualKeyboard")}} 对象。通过该对象，你可以选择退出自动虚拟键盘行为、以编程方式显示或隐藏虚拟键盘，以及获取虚拟键盘的当前位置和大小。
```

**Review by jasonren0403 — CHANGES_REQUESTED** [review]
有些小问题

**Review by jasonren0403 — APPROVED** [review]
👍

**Comment by T34-active** [comment]
@jasonren0403 Thank you!

**Pull request merged**